diff --git a/src/HIAHKernel/Core/HIAHKernel.m b/src/HIAHKernel/Core/HIAHKernel.m
index 5c1bd1c..a5c5c1c 100644
--- a/src/HIAHKernel/Core/HIAHKernel.m
+++ b/src/HIAHKernel/Core/HIAHKernel.m
@@ -32,7 +32,7 @@ static void extensionStartedCallback(CFNotificationCenterRef center,
   NSFileManager *fm = [NSFileManager defaultManager];
   NSURL *groupURL =
       [fm containerURLForSecurityApplicationGroupIdentifier:
-              @"group.com.aspauldingcode.HIAHDesktop"];
+              @"group.com.aspauldingcode.Wawona"];
   NSString *logPath = nil;
   if (groupURL) {
     logPath = [[groupURL.path stringByAppendingPathComponent:@"HIAHKernel.log"]
